pkg/storage/utils/decomposedfs/tree/tree.go

```diff
@@ -48,7 +48,10 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-var tracer trace.Tracer
+var (
+	tracer           trace.Tracer
+	ErrSizeUnlimited = errors.New("unlimited")
```

Comment:
I know it's catched shortly after, but can't still have a more describing error like "quota unlimited"?